packages/core/src/devtools/injectDevTools.ts

```diff
@@ -0,0 +1,30 @@
+import { SEQUENCE_PREFIX } from '../constants';
+import { getDebugTree } from './getDebugTree';
+
+export function injectDevTools(document: Document) {
+  const window = document.defaultView;
+  if (!window || Object.prototype.hasOwnProperty.call(window, '__GRIFFEL_DEVTOOLS__')) {
```

Comment:
```suggestion
  if (!window || window.__GRIFFEL_DEVTOOLS__)) {
```

Should it be really `.hasOwnProperty`?